ansible/inventory

```diff
@@ -0,0 +1,5 @@
+[sandbox]
+sandbox.isic-archive.com ansible_ssh_user=ubuntu ansible_python_interpreter=/usr/bin/python2.7
+
+[sandbox:vars]
+bind_node_modules=false
```

Comment:
This is not getting set properly, as this was bound on the provisioned machine. Why do we need to put this as a variable, if we have our own playbook for the sandbox (or more generally, EC2-deployed) machine?

For now, I'm manually removing this on the provisioned machine.